node_modules/gh-orgunit/lib/api.js

```diff
@@ -69,6 +72,20 @@ var createOrgUnit = module.exports.createOrgUnit = function(ctx, appId, displayN
         validator.check(parentId, {'code': 400, 'msg': 'A valid parent organisational unit id must be provided'}).isInt();
         parentId = GrasshopperUtil.getNumberParam(parentId);
     }
+    if (metadata) {
+        if (_.isString(metadata)) {
+            try {
```

Comment:
Maybe we should update the JSDocs and specify that both `string` and `object` are accepted